### ISSUE TITLE
Calling @PreCallBack twice

### ISSUE THREAD

**Issue by wquintanilhadasilva** [alisson-gomesc/jkniv#6]
In save operations, the framework is calling @PrePersist  methods twice.

**Comment by alisson-gomesc** [comment]
Show me your class please.

**Comment by wquintanilhadasilva** [comment]
Sorry, I edited the annotation name. The correct name is @PreCallBack :-)

Method in the business class
```

    @PreCallBack(scope = CallbackScope.ADD)
    public void generateId() {
	if (!this.prefix.equals(EntityPrefix.NONE.toString())) {
	    this._id = this.prefix + DateTimeUtils.nowUTCTimeStampString();
	}
    }    

```

Call the Repository in the service:

```
...
    T item = this.repository.add(item)
...
```

**Comment by alisson-gomesc** [comment]
Sorry about delay, happy new year :)

This error is present in the versions `0.6.0.M43` and `0.6.0.M44`, the version `0.6.0.M45` was released fixing that. An unit test case  was wrote to not repeat again.

```xml
<dependency>
    <groupId>net.sf.jkniv</groupId>
    <artifactId>jkniv-whinstone-couchdb</artifactId>
    <version>0.6.0.M45</version>
</dependency>
```

**Comment by alisson-gomesc** [comment]
Fixed `0.6.0.M45` version.
